heartbeat/monitors/active/script/script.go

```diff
@@ -69,12 +72,49 @@ func create(name string, cfg *common.Config) (js []jobs.Job, endpoints int, err
 		}
 		stdin.Close()
 
-		stdoutResults := decodePipe(stdout)
 		stderrResults := decodePipe(stderr)
 
-		eventext.MergeEventFields(event, map[string]interface{}{
-			"script.stdout": stdoutResults,
-			"script.stderr": stderrResults,
+		var stdoutResults []string
+		var rawResults map[string]interface{}
+		for _, line := range decodePipe(stdout) {
+			if s, ok := line.(string); ok {
+				logp.Warn("STRING HERE")
+				stdoutResults = append(stdoutResults, s)
+			} else if r, ok := line.(Results); ok {
+				status := "down"
+				rawResults = r.Raw
+
+				if len(r.Journeys) > 0 {
+					journey := r.Journeys[0]
+					if journey.error != nil {
+						status = "down"
```

Comment:
should we do this from Node itself? feels cleaner? . each journey will have a status associated with it.